src/cascadia/TerminalControl/SearchBoxControl.h

```diff
@@ -14,33 +14,45 @@ Author(s):
 --*/
 
 #pragma once
-#include "winrt/Windows.UI.Xaml.h"
-#include "winrt/Windows.UI.Xaml.Controls.h"
-
 #include "SearchBoxControl.g.h"
 
 namespace winrt::Microsoft::Terminal::Control::implementation
 {
     struct SearchBoxControl : SearchBoxControlT<SearchBoxControl>
     {
+        static constexpr int32_t MaximumTotalResultsToShowInStatus = 999;
+        static constexpr std::wstring_view TotalResultsTooHighStatus = L"999+";
+        static constexpr std::wstring_view CurrentIndexTooHighStatus = L"?";
+        static constexpr std::wstring_view StatusDelimiter = L"/";

```

Comment:
BTW, unrelated to this PR, I think we should technically localize these generated things as well. I don't want to say that this is important, I think most people will understand "x/y" to mean "x out of y", but at least technically speaking not all languages have given "/" the meaning of "out of" yet and even worse, some write it in reverse where you say y first and then x.

Reply:
This is actually a good note for _this_ PR